sets.go

```diff
@@ -146,6 +128,48 @@ func (db *RoseDB) SCard(key []byte) int {
 	return db.setIndex.trees[string(key)].Size()
 }
 
+// SDiff returns the members of the set difference between the first set and
+// all the successive sets. Returns error if no key is passed as a parameter.
+func (db *RoseDB) SDiff(keys ...[]byte) ([][]byte, error) {
+	db.setIndex.mu.RLock()
+	defer db.setIndex.mu.RUnlock()
+	if len(keys) == 0 {
+		return nil, ErrWrongNumberOfArgs
+	}
+	if len(keys) == 1 {
+		return db.sMembers(keys[0])
+	}
+
+	firstSet, err := db.sMembers(keys[0])
+	if err != nil {
+		return nil, err
+	}
+	successiveSet := make(map[uint64]struct{})
+	for _, key := range keys[1:] {
+		members, err := db.sMembers(key)
+		if err != nil {
+			return nil, err
+		}
+		for _, k := range members {
```

Comment:
the variable `k` seems to represent `key`,  `mem` or `value` is better here.

Reply:
Done in d49fdfb.